src/frontend/src/handler/mod.rs

```diff
@@ -134,17 +134,11 @@ pub async fn handle(
                 )
                 .into());
             }
-            if if_not_exists {
```

Comment:
wonder why we had this in the first place .. 🤔

Reply:
🤔Same